Ignition Edifice: roadmap, installation, info (#95)
ignitionrobotics/docs: edifice/install_ubuntu.md

# Binary Installation on Ubuntu

> Dome binaries are provided for Ubuntu Bionic and Focal. All of the Dome

Reviewer: ```suggestion
Edifice nightlies are provided for Ubuntu Bionic and Focal. All of the Edifice
```

binaries are hosted in the osrfoundation repository. To install all of them,

> the metapackage `ignition-dome` can be installed:

Reviewer: ```suggestion
the metapackage `ignition-edifice` can be installed:
```

```bash

> sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'

Reviewer: ```suggestion
sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-nightly `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-nightly.list'
```

sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-nightly `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-nightly.list'
wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
sudo apt-get update

> sudo apt-get install ignition-dome

Reviewer: ```suggestion
sudo apt-get install ignition-edifice
```

All libraries should be ready to use and the `ign gazebo` app ready to be executed.

Head back to the [Getting started](/docs/all/get_started)
page to start using Ignition!

## Uninstalling binary install

If you need to uninstall Ignition or switch to a source-based install once you
have already installed the library from binaries, run the following command:

```bash

> sudo apt remove ignition-dome && sudo apt autoremove

Reviewer: ```suggestion
sudo apt remove ignition-edifice && sudo apt autoremove
```
